src/user_content.py

```diff
@@ -37,6 +37,7 @@ def get(self, owner, repo, tag, name, path):
 
     config_map = {}
     for dependency in dependencies:
+      dependency['owner'] = dependency['owner'].replace('git://github.com/', '')
```

Comment:
Did we decide that this was necessary? Weren't we going to try and fix this in analysis?